[OSDOCS#17175]: Make changes as per the errors and warnings for CQA-3 (#114275)
openshift/openshift-docs: modules/compliance-filtering-results.adoc

[id="filtering-compliance-check-results_{context}"]
= Filters for compliance check results

[role="_abstract"]

> By default, the `ComplianceCheckResult` objects are labeled with several useful labels that allow you to query the checks and decide on the next steps after the results are generated.

Reviewer: ```suggestion
By default, the `ComplianceCheckResult` objects are labeled with several useful labels that you can use to query the checks and decide on the next steps after the results are generated.
```
Avoid anthropomorphic language ("allow you").

[id="filtering-compliance-check-results_{context}"]
= Filters for compliance check results

[role="_abstract"]

> By default, the `ComplianceCheckResult` objects are labeled with several useful labels that you can use to query the checks and decide on the next steps after the results are generated.

Reviewer 2: ```suggestion
You can use the labels in the `ComplianceCheckResult` objects to query the checks and decide on the next steps after the results are generated.
```
